src/dropdown/dropdown.ts

```diff
@@ -65,6 +93,11 @@ export class NgbDropdown {
    */
   close(): void {
     if (this._open) {
+      if (this.container === 'body') {
+        this._elementRef.nativeElement.appendChild(this._dropdownMenu);
+        window.document.querySelector(this.container).classList.remove('open');
```

Comment:
Could we go through a renderer here? More generally I'm a bit nervous about the amount of DOM manipulation that is going on here. I would definitively like to make sure that we are still properly rendered on the server-side with Universal.